padout should be compiled using the native compiler but isn't. 
Disable this piece of documentation for now.

RP

Index: Linux-PAM-1.0.2/doc/specs/Makefile.am
===================================================================
--- Linux-PAM-1.0.2.orig/doc/specs/Makefile.am	2008-11-04 21:06:23.000000000 +0000
+++ Linux-PAM-1.0.2/doc/specs/Makefile.am	2008-11-04 21:07:06.000000000 +0000
@@ -2,21 +2,8 @@
 # Copyright (c) 2005, 2006 Thorsten Kukuk <kukuk@suse.de>
 #
 
-CLEANFILES = draft-morgan-pam-current.txt *~
+CLEANFILES = *~
 
-EXTRA_DIST = draft-morgan-pam.raw std-agent-id.raw rfc86.0.txt
+EXTRA_DIST = std-agent-id.raw rfc86.0.txt
 
-draft-morgan-pam-current.txt: padout draft-morgan-pam.raw
-	./padout < $(srcdir)/draft-morgan-pam.raw > draft-morgan-pam-current.txt
-
-AM_YFLAGS = -d
-
-BUILT_SOURCES = parse_y.h
-
-noinst_PROGRAMS = padout
-
-padout_SOURCES = parse_l.l parse_y.y
-
-padout_LDADD = @LEXLIB@
-
-doc_DATA = draft-morgan-pam-current.txt rfc86.0.txt
+doc_DATA = rfc86.0.txt
